cmd/containerd-stargz-grpc/main.go

```diff
@@ -185,6 +197,18 @@ func main() {
 	log.G(ctx).Info("Exiting")
 }
 
+// isFusePthEnable prevents users from enabling passthrough mode on unsupported kernel versions
+func isFusePthEnable() (bool, error) {
+	cmd := exec.Command("sh", "-c", "grep 'CONFIG_FUSE_PASSTHROUGH=y' /boot/config-$(uname -r)")
```

Comment:
nit: Let's try not to rely on the shell commands for now. Instead of having this check, let's put a document about how to check if passthrough is a supported on the node (maybe in the following PRs)